ui2/src/pages/clearing/Member.tsx

```diff
@@ -0,0 +1,203 @@
+
+  const pendingMarginCalcs = useStreamQueries(MarginCalculation)
+    .contracts.filter(mc => mc.payload.customer === customer)
+    .reverse();
+  const failedMarginCalcs = useStreamQueries(RejectedMarginCalculation)
+    .contracts.filter(mc => mc.payload.customer === customer)
+    .reverse();
+  const fulfilledMarginCalcs = useStreamQueries(FulfilledMarginCalculation)
+    .contracts.filter(mc => mc.payload.customer === customer)
+    .reverse();
+
+  const pendingMTMCalcs = useStreamQueries(MarkToMarketCalculation)
+    .contracts.filter(mc => mc.payload.customer === customer)
+    .reverse();
+  const failedMTMCalcs = useStreamQueries(RejectedMarkToMarketCalculation)
+    .contracts.filter(mc => mc.payload.customer === customer)
+    .reverse();
+  const fulfilledMTMCalcs = useStreamQueries(FulfilledMarkToMarketCalculation)
+    .contracts.filter(mc => mc.payload.customer === customer)
+    .reverse();
+
+  const handleMTMRetry = async (cid: ContractId<RejectedMarkToMarketCalculation>) => {
+    const choice = RejectedMarkToMarketCalculation.RejectedMarkToMarketCalculation_Retry;
+    await ledger.exercise(choice, cid, {});
+  };
+  const handleMTMCancel = async (cid: ContractId<RejectedMarkToMarketCalculation>) => {
+    const choice = RejectedMarkToMarketCalculation.RejectedMarkToMarketCalculation_Cancel;
+    await ledger.exercise(choice, cid, {});
+  };
+
+  const handleMarginRetry = async (cid: ContractId<RejectedMarginCalculation>) => {
+    const choice = RejectedMarginCalculation.RejectedMarginCalculation_Retry;
+    await ledger.exercise(choice, cid, {});
+  };
+  const handleMarginCancel = async (cid: ContractId<RejectedMarginCalculation>) => {
+    const choice = RejectedMarginCalculation.RejectedMarginCalculation_Cancel;
+    await ledger.exercise(choice, cid, {});
+  };
+
+  return (
+    <div className="assets">
+      <Tile header={<h2>Actions</h2>}>
+        <MarginCallModal services={services} member={customer} />
+        <MTMCalculationModal services={services} member={customer} />
+      </Tile>
+      <Tile header={<h2>Standing</h2>}>
+        <b>Margins:</b> {!!standing && standing?.payload.marginSatisfied ? 'Yes' : 'No'}
+        <br />
+        <b>MTM:</b> {!!standing && standing?.payload.mtmSatisfied ? 'Yes' : 'No'}
+        <br />
+        <b>Margin Amount:</b> {formatter.format(marginAmount)}
+        <br />
+        <b>Clearing Amount:</b> {formatter.format(clearingAmount)}
+      </Tile>
+      <Header as="h2">Margin Calculations</Header>
+      <Header as="h3">Curent Margin Calculations</Header>
+      <StripedTable
+        headings={['Time', 'Target Amount', 'Account']}
+        rows={pendingMarginCalcs.map(mc => {
+          return [
+            <>{mc.payload.calculationTime}</>,
+            <>{formatter.format(Number(mc.payload.targetAmount))}</>,
+            <>{mc.payload.accountId.label}</>,
+          ];
+        })}
+      />
+      <Header as="h3">Failed Margin Calculations</Header>
```

Comment:
maybe only show these if there are any? Unless this is common or expected?

Reply:
I just tried this and it gets kinda weird because Failed Margin Calculations have a retry button, which when you press it (and it still fails), they'll briefly turn into a Pending calculation and then back to a failed one, so there's a bunch of weird flashing.

Either way I think page and the rest of the clearing ui will need some design help from Jennifer